src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -252,6 +252,24 @@ public void SetDirty(TimeSpan? maxDelay)
             }
         }
 
+        public void ProcessPushNotification(PushNotification notification, TimeSpan? maxDelay)
+        {
+            
+            if (notification == null) 
+            { 
+                throw new ArgumentNullException(nameof(notification)); 
+            }
+            if (notification.SyncToken == null || notification.EventType == null || notification.ResourceUri == null)

```

Comment:
Nit: leave a line after the closing bracket `}`. Can you update this in other places too?